File: TestApplications/Playframework_TestApp/app/models/Bid.java
package models;


import play.db.ebean.*;
import play.db.ebean.Model.Finder;

import javax.persistence.*;

@Entity
public class Bid extends Model{
	@Id
	@Column(name="bidID")
	private int bidID;
	@ManyToOne(cascade = CascadeType.ALL)
	@JoinColumn(name="itemno")
	private Item itemno;
	@ManyToOne(cascade = CascadeType.ALL)
	@JoinColumn(name="userID")
	private User userID;
	@Column(name="value")
	private int value;
	@Column(name="username")
	private String username;
	
	public Bid(int itemno, long userID, int value, String username) {
		this.itemno = Item.find.byId(itemno);
		this.userID = User.find.byId(userID);
		this.value = value;
		this.username = username;
	}

	public static Finder<Long,Bid> find = new Finder<Long,Bid>(Long.class, Bid.class);
	
	public boolean add() {
		this.save();
		return this.bidID > 0;
	}
	
	public int getBidID() {
		return bidID;
	}

	public Item getItemno() {
		return itemno;
	}

	public User getUserID() {
		return userID;
	}

	public int getValue() {
		return value;
	}

	public String getUsername() {
		return username;
	}

	public void setBidID(int bidID) {
		this.bidID = bidID;
	}

	public void setItemno(Item itemno) {
		this.itemno = itemno;
	}

	public void setUserID(User userID) {
		this.userID = userID;
	}

	public void setValue(int value) {
		this.value = value;
	}

	public void setUsername(String username) {
		this.username = username;
	}
}

File: TestApplications/SignalR_Testapp/SignalR_Testapp/Models/Bid.cs
namespace SignalR_Testapp.Models
{
    public class Bid
    {
        private long _bidID;
        private long _itemno;
        private long _userID;
        private long _value;
        private string _username;

        public long bidID { get { return this._bidID; } set { this._bidID = value; } }
        public long itemno { get { return this._itemno; } set { this._itemno = value; } }
        public long userID { get { return this._userID; } set { this._userID = value; } }
        public long value { get { return this._value; } set { this._value = value; } }
        public string username { get { return this._username; } set { this._username = value; } }
    }
}